src/com/javabasic/lesson15/interfaces/Converter.java

```diff
@@ -0,0 +1,7 @@
+package com.javabasic.lesson15.interfaces;
+
+public interface Converter {
+
+    void convert(double degree);
```

Comment:
It's better to return value from this method. Imagine you need to use it further in the code.

Reply:
done